_posts/2021-04-14-cockpit-242.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cockpit 242
+author: marusak
+date: 2021-04-14
+tags: cockpit linux
+slug: cockpit-242
+category: release
+summary: Cockpit ...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 242.
+
+
+## Support for pages built with snowpack
+
+[Snowpack](https://www.snowpack.dev/) is a web project build system alternative to [webpack](https://webpack.js.org/). When using it to build a Cockpit page that uses an [npm module](https://www.npmjs.com/) containing a `@`, such as [@patternfly](https://www.npmjs.com/package/@patternfly/react-core), the page [could not be loaded](https://github.com/cockpit-project/cockpit/issues/15288). This has been fixed now. Note that there is not yet a fully working and supported example of a snowpack-built Cockpit project, just some [initial experimentation](https://piware.de/post/2021-02-05-snowpack/).
+
+
+## Machines: Split out to separate project
+
+The Machines page is by far the largest Cockpit component. It got split out to the standalone [cockpit-machines project](https://github.com/cockpit-project/cockpit-machines/) to make it easier and faster to develop. The same split happened in [Fedora](https://bodhi.fedoraproject.org/updates/?packages=cockpit-machines) and [Debian](https://tracker.debian.org/pkg/cockpit-machines).
+
+## Cockpit website refreshed
+
+It includes:
```

Comment:
You refer to an "it" which has so far only been introduced in the heading, which is not our usual style.  Add a short text about how we have a new website.